pkg/kubelet/cm/cpuset/cpuset.go

```diff
@@ -19,25 +19,22 @@ package cpuset
 import (
 	"bytes"
 	"fmt"
-	"os"
 	"reflect"
 	"sort"
 	"strconv"
 	"strings"
-
-	"k8s.io/klog/v2"
 )
 
-// Builder is a mutable builder for CPUSet. Functions that mutate instances
+// builder is a mutable builder for CPUSet. Functions that mutate instances
 // of this type are not thread-safe.
-type Builder struct {
+type builder struct {
 	result CPUSet
 	done   bool
 }
 
-// NewBuilder returns a mutable CPUSet builder.
-func NewBuilder() *Builder {
-	return &Builder{
+// newBuilder returns a mutable CPUSet builder.
+func newBuilder() *builder {
```

Comment:
I don't actually object to this, but it feels unnecessary - every place that uses it could just be:

```
s := New()
s.add(x)
s.add(y)
```

or even

```
s := New()
s.elems[x] = struct{}{}
```

It's only mutable from within this package, and having the builder doesn't make it much safer, and only slightly more obvious if someone was trying to mutate it when they shouldn't.

Reply:
Good idea. Thanks Tim for the thorough reivew.

Done.